samples/containers/encrypted/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+test_mode=$1
+run_mode=$2
+
+set -e
+
+if [[ "$test_mode" == "clean" ]]; then
+    make clean
+elif [[ "$test_mode" == "init" ]]; then
+    make
+elif [[ "$test_mode" == "run" ]]; then
+    make "$run_mode"-verity
+    # TODO: This doesn't work
```

Comment:
is there an issue for this TODO or is it going to be added to an existing issue?

Reply:
I'm going to create an issue for this, and for the openmp and nodejs broken samples.

Reply:
https://github.com/lsds/sgx-lkl/issues/831